server/settings.go

```diff
@@ -49,3 +49,42 @@ func (p *Plugin) settingsNotifications(header *model.CommandArgs, instanceID, ma
 
 	return p.responsef(header, "Settings updated. Notifications %s.", notifications)
 }
+
+func (p *Plugin) settingsWatching(header *model.CommandArgs, instanceID, mattermostUserID types.ID, connection *Connection, args []string) *model.CommandResponse {
+	const helpText = "`/jira watching watching [value]`\n* Invalid value. Accepted values are: `on` or `off`."
+
+	if len(args) != 2 {
+		return p.responsef(header, helpText)
+	}
+
+	var value bool
+	switch args[1] {
+	case settingOn:
+		value = true
+	case settingOff:
+		value = false
+	default:
+		return p.responsef(header, helpText)
+	}
+
+	if connection.Settings == nil {
+		connection.Settings = &ConnectionSettings{}
+	}
+	connection.Settings.Watching = value
+	if err := p.userStore.StoreConnection(instanceID, mattermostUserID, connection); err != nil {
+		p.errorf("settingsWatching, err: %v", err)
+		p.responsef(header, "Could not store new settings. Please contact your system administrator. error: %v", err)
+	}
+
+	// send back the actual value
+	updatedConnection, err := p.userStore.LoadConnection(instanceID, mattermostUserID)
+	if err != nil {
+		return p.responsef(header, "Your username is not connected to Jira. Please type `jira connect`. %v", err)
+	}
+	watching := settingOff
+	if updatedConnection.Settings.Watching {
+		watching = settingOn
+	}
+
+	return p.responsef(header, "Settings updated. Watching %s.", watching)
```

Comment:
@mickmister, do you think it is necessary to check the update after the connection is stored? If the `StoreConnection()` fails, an error response is sent. Do we really need to read the value back to give the response that settings have been updated?  
```suggestion
	return p.responsef(header, "Settings updated. Watching %s.", value)
```

Reply:
@jfrerich I've pushed a new commit for this change, could you please check it?